### PR TITLE
Fixes/issue 281 editor styles

### PR DESCRIPTION
**Pull request by laurelfulford**
Doing a first pass of editor-styles.css. See #281.

### CODE REVIEW

**Comment by josephfusco** [comment]
Looks good to me!

**Comment by josephfusco** [comment]
We will need to remind any existing PRs to also update any applicable styles in editor-style.css.

File: assets/css/editor-style.css [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+
+body,
+button,
+input,
+select,
+textarea {
+	font-family: "Libre Franklin", "Helvetica Neue", helvetica, arial, sans-serif;
+	font-size: 16px;
+	font-size: 1rem;
+	font-weight: normal;
```
Comment (ntwb) [inline assets/css/editor-style.css]: Numeric font values, `400`, not `normal`

File: assets/css/editor-style.css [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+}
+
+li > ul,
+li > ol {
+	margin-bottom: 0;
+	margin-left: 1.5em;
+}
+
+dt {
+	font-weight: bold;
```
Comment (ntwb) [inline assets/css/editor-style.css]: Numeric font values, `700`, not `bold`

File: assets/css/editor-style.css [hunk 3]
```diff
@@ -0,0 +1,415 @@
...
+
+.wp-caption .wp-caption-text {
+	margin: 0.8075em 0;
+}
+
+/**
+ * 6.0 - Galleries
+ */
+
+ .gallery {
```
Comment (ntwb) [inline assets/css/editor-style.css]: Remove the space before the period at the start of the line

File: functions.php [hunk 4]
```diff
@@ -87,6 +87,12 @@ function twentyseventeen_setup() {
...
 	) );
+
+	/*
+	 * This theme styles the visual editor to resemble the theme style,
+	 * specifically font, colors, and column width.
+ 	 */
+	add_editor_style( array( 'assets/css/editor-style.css', twentyseventeen_fonts_url() ) );
 }
 endif;
 add_action( 'after_setup_theme', 'twentyseventeen_setup' );
```
Comment (ntwb) [inline functions.php:98]: Add a blank newline at the end of the file please

File: assets/css/editor-style.css [hunk 5]
```diff
@@ -0,0 +1,415 @@
...
+	font-size: 1rem;
+	font-weight: normal;
+	line-height: 1.66;
+}
+
+h1, h2, h3, h4, h5, h6 {
+	clear: both;
+	line-height: 1.4;
+	margin: 0 0 0.75em;
+	padding: 1.5em 0 0 0;
```
Comment (ntwb) [inline assets/css/editor-style.css]: Use shorthand properties, this `padding: 1.5em 0 0;`, not `padding: 1.5em 0 0 0;`

File: assets/css/editor-style.css [hunk 6]
```diff
@@ -0,0 +1,415 @@
...
+}
+
+big {
+	font-size: 125%;
+}
+
+blockquote, q {
+	quotes: "" "";
+}
+blockquote:before,
```
Comment (ntwb) [inline assets/css/editor-style.css:187]: Expected blank newline before this selector

File: assets/css/editor-style.css [hunk 7]
```diff
@@ -0,0 +1,415 @@
...
+select,
+textarea {
+	font-family: "Libre Franklin", "Helvetica Neue", helvetica, arial, sans-serif;
+	font-size: 16px;
+	font-size: 1rem;
+	font-weight: normal;
+	line-height: 1.66;
+}
+
+h1, h2, h3, h4, h5, h6 {
```
Comment (ntwb) [inline assets/css/editor-style.css]: Expected each of these on their own line:

``` css
h1,
h2,
h3,
h4,
h5,
h6 {
```

File: assets/css/editor-style.css [hunk 8]
```diff
@@ -0,0 +1,415 @@
...
+	font-size: 0.9375rem;
+	font-weight: 800;
+}
+
+p {
+	margin: 0 0 1.5em;
+	padding: 0;
+}
+
+dfn, cite, em, i {
```
Comment (ntwb) [inline assets/css/editor-style.css]: Expected newline after each comma:

``` css
dfn,
cite,
em,
i {
```

File: assets/css/editor-style.css [hunk 9]
```diff
@@ -0,0 +1,415 @@
...
+	font-size: 15px;
+	font-size: 0.9375rem;
+}
+
+abbr, acronym {
+	border-bottom: 1px dotted #666;
+	cursor: help;
+}
+
+mark, ins {
```
Comment (ntwb) [inline assets/css/editor-style.css]:
``` css
mark,
ins {
```

File: assets/css/editor-style.css [hunk 10]
```diff
@@ -0,0 +1,415 @@
...
+	padding: 1.6em;
+}
+
+code, kbd, tt, var {
+	font-family: Monaco, Consolas, "Andale Mono", "DejaVu Sans Mono", monospace;
+	font-size: 15px;
+	font-size: 0.9375rem;
+}
+
+abbr, acronym {
```
Comment (ntwb) [inline assets/css/editor-style.css]:
``` css
abbr, 
acronym {
```

File: assets/css/editor-style.css [hunk 11]
```diff
@@ -0,0 +1,415 @@
...
+	font-size: 15px;
+	font-size: 0.9375rem;
+	line-height: 1.6;
+	margin-bottom: 1.6em;
+	max-width: 100%;
+	overflow: auto;
+	padding: 1.6em;
+}
+
+code, kbd, tt, var {
```
Comment (ntwb) [inline assets/css/editor-style.css]:
``` css
code, 
kbd, 
tt, 
var {
```

File: assets/css/editor-style.css [hunk 12]
```diff
@@ -0,0 +1,415 @@
...
+mark, ins {
+	background: #eee;
+	text-decoration: none;
+}
+
+big {
+	font-size: 125%;
+}
+
+blockquote, q {
```
Comment (ntwb) [inline assets/css/editor-style.css]:
``` css
blockquote,
q {
```

**Review by ntwb — CHANGES_REQUESTED** [review]
@laurelfulford Added CSS coding standards notes

File: functions.php (hunk 4 above)
Comment (laurelfulford) [reply functions.php:98]: Thanks for the code review, @ntwb!

Everything should be addressed, except this one - it looks like both the functions.php and the editor-style.css files already already end with a new line in this commit. I might be misunderstanding - please let me know if this is what you meant, or something else. Thanks! 🙂

**Pull request merged**